common/check_rule_counts

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
Please use the new copyright notice (e.g. from `check_storage_space`, but without the phrase ‘for the benefit of the ATLAS collaboration’).